### PR TITLE
Add storybook

### PR DESCRIPTION
**Pull request by Hacksore**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,9 +1,8 @@
 {
 	"name": "skinview3d",
-	"version": "2.0.0-alpha.5",
+	"version": "2.0.1",
 	"description": "Three.js powered Minecraft skin viewer",
 	"main": "libs/skinview3d.js",
-	"type": "module",
```
Comment (Hacksore) [inline package.json:6]: @yushijinhun @g-plane storybook was not a fan of this. Does removing it cause any concern or issues?

Comment (g-plane) [reply package.json:6]: Is removing this necessary?

Comment (Hacksore) [reply package.json:6]: Storybook throws this error if I don't remove it.

```log
WARN Unable to close preview build!
ERR! Error [ERR_REQUIRE_ESM]: Must use import to load ES Module: D:\Code\Gaming\skinview3d\.storybook\main.js
ERR! require() of ES modules is not supported.
ERR! require() of D:\Code\Gaming\skinview3d\.storybook\main.js from D:\Code\Gaming\skinview3d\node_modules\@storybook\core\dist\server\utils\server-require.js is an ES module file as it is a .js file whose nearest parent package.json contains "type": "module" which defines all .js files in that package scope as ES modules.
ERR! Instead rename main.js to end in .cjs, change the requiring code to use import(), or remove "type": "module" from D:\Code\Gaming\skinview3d\package.json.
```

File: package.json [hunk 2]
```diff
@@ -1,9 +1,8 @@
 {
 	"name": "skinview3d",
-	"version": "2.0.0-alpha.5",
+	"version": "2.0.1",
```
Comment (yushijinhun) [inline package.json:3]: Why change the version?

File: package.json (hunk 1 above)
Comment (yushijinhun) [reply package.json:6]: Resolved in https://github.com/bs-community/skinview3d/commit/86731e23b9b1cc8f4855471f179080a1bdd42843

File: package.json (hunk 2 above)
Comment (Hacksore) [reply package.json:3]: I figured storybook being added could then be tracked to a tag version. If you don't agree with this approach I can revert it.

Comment (yushijinhun) [reply package.json:3]: Please revert it. I don't think it's a good idea to bump version number in PR.

Releasing should be done by running `npm version <new_version>`, which will create a new commit & a tag. Then you need to push the commit to master and run `npm publish`.

File: package.json [hunk 3]
```diff
@@ -14,8 +14,10 @@
 		"dev:watch:modules": "tsc -w --declaration --sourceMap --outDir libs -p .",
 		"dev:watch:bundles": "rollup -w -c",
 		"dev:serve": "ws",
+		"storybook": "start-storybook -s ./public -p 6006",
 		"dev": "npm-run-all --parallel dev:watch:modules dev:watch:bundles dev:serve",
-		"prepublishOnly": "npm run clean && npm run build"
+		"prepublishOnly": "npm run clean && npm run build",
+		"build-storybook": "build-storybook"
```
Comment (yushijinhun) [inline package.json:20]: What is this script used for?

File: stories/index.stories.js [hunk 4]
```diff
@@ -0,0 +1,100 @@
...
+	};
+
+	const capeUrl = radios("Cape Textures", capeOptions, "none");
+	viewer.loadSkin(`texture/${skinUrl}.png`);
+
+	if (capeUrl === "none") {
+		viewer.loadCape(null);
+	}
+
+	viewer.loadCape(`texture/${capeUrl}.png`);
```
Comment (yushijinhun) [inline stories/index.stories.js:97]: This should be in else block since capeUrl can be "none" here.

File: package.json (hunk 3 above)
Comment (Hacksore) [reply package.json:20]: It's part of storybook cli which can build static output, it's not being used at the moment. 

I'd like to in the future have storybook built as static for the gh-pages branch. That way people have a nice sandbox to explore the libs feature set.

**Comment by Hacksore** [comment]
Also might want to point out that we need to drop the following things at some point.
`local-web-server` as a devDep since we don't need a local web server anymore and the whole examples folder. 

However, this can be done at a later time. I'll want to make sure we have feature parity from the old example page.

File: package.json (hunk 3 above)
Comment (yushijinhun) [reply package.json:20]: Sounds great. This process can be automated using GitHub Actions in the future.

Is dist/ directory also for this purpose? I think `npm run build-storybook` is no different from `npx build-storybook`, and we don't need them currently.

Comment (Hacksore) [reply package.json:20]: I's say for the scope of this PR let's not worry about the built output of storybook.

Do you want me to remove this script?

**Comment by yushijinhun** [comment]
@Hacksore `examples/index.html` can die now. Also, I suggest move `stories/` to `examples/stories/` and `public/texture/` to `examples/textures/`. We also need a `README.md` in `examples/textures/` to state the copyright ownership of the textures.

File: package.json (hunk 3 above)
Comment (yushijinhun) [reply package.json:20]: Yeah, plz remove it.

**Pull request merged**